### PR TITLE
Missing Translations and Notification Bug fix

### LINKED ISSUE #216: Missing language strings for translations
**Describe the bug**
Some strings has no language translation availability. I listed some of these which I found.

**To Reproduce**
- Missing string "Put on repeat list" for translation

1. Go to 'https://www.mikazuki.moe/application/#/aniList/completed'
2. Mouse over on 'Put on repeat list'
image

*  Missing string "Revert to airing list" for translation
1. Go to 'https://www.mikazuki.moe/application/#/aniList/paused' & 'https://www.mikazuki.moe/application/#/aniList/dropped'
2. Mouse over on 'Revert to airing list'
image

* Missing string "You're currently not logged in! Please do so by clicking on the "Login" button below." translation
after logged of > settings > AniList > 
image

* 'notifications.aniList.' Possible bug (might not english translation)
Go random anime i.e (https://www.mikazuki.moe/application/#/detail-view/249)
Change media status in these settings: Paused|Planning|Repeating|Dropped can cause 
image

**Desktop:**
 - OS: [Windows 10 2004 x64] 
 - Browser [Microsoft Edge]
 - Version [87.0.664.60 (64 bit)]

### PR DESCRIPTION
**Pull request by NicoAiko**
Fixes #216
Adds missing strings to translation files
Fixes bug where notification message text is faulty

### CODE REVIEW

File: src/assets/locales/ar.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
     "search": "بحث",
     "sort": "فرز وصفى",
     "toPageTop": "القفز إلى الأعلى",
-    "filter": "فرز"
+    "filter": "فرز",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/ar.json:21]: @Arabic translation team: Please add a **suggestion** here

File: src/assets/locales/ar.json [hunk 2]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/ar.json:22]: @Arabic translation team: Please add a **suggestion** here

File: src/assets/locales/ar.json [hunk 3]
```diff
@@ -195,7 +197,8 @@
           "showExplicitContentHint": "هذا الخيار سيحدد ظهور محتوي لـ18+ من عدمه.",
           "userTitleLanguage": "لغة الاسم",
           "userTitleLanguageHint": "هذا الخيار يحدد اي عنوان سيستخدم لعرض اسم المحتوي."
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/ar.json:201]: @Arabic translation team: Please add a **suggestion** here

File: src/assets/locales/es_MX.json [hunk 4]
```diff
@@ -17,7 +17,9 @@
     "save": "Guardar",
     "search": "Buscar",
     "sort": "Ordenar",
-    "toPageTop": "Ir al inicio de la página"
+    "toPageTop": "Ir al inicio de la página",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/es_MX.json:21]: @Spanish translation team: Please add a **suggestion** here

File: src/assets/locales/es_MX.json [hunk 5]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/es_MX.json:22]: @Spanish translation team: Please add a **suggestion** here

File: src/assets/locales/es_MX.json [hunk 6]
```diff
@@ -117,7 +119,7 @@
         "notYetAired": "Aún no emitido",
         "nextAiringEpisode": "El episodio {0} se emitirá el {1}",
         "scoreDialog": {
-         "title": "Cambiar puntuación"
+          "title": "Cambiar puntuación"
```
Comment (NicoAiko) [inline src/assets/locales/es_MX.json:122]: @Spanish translation team: Please add a **suggestion** here

File: src/assets/locales/es_MX.json [hunk 7]
```diff
@@ -195,7 +197,8 @@
           "subHeadline": "Los cambios en esta preferencia podrían tardar un tiempo hasta entrar en efecto.",
           "userTitleLanguage": "Lenguaje de los títulos",
           "userTitleLanguageHint": "Esta preferencia define qué lenguaje se usará para mostrar los nombres de los títulos."
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/es_MX.json:201]: @Spanish translation team: Please add a **suggestion** here

File: src/assets/locales/fr.json [hunk 8]
```diff
@@ -17,7 +17,9 @@
     "save": "Sauvegarder",
     "search": "Chercher",
     "sort": "Trier",
-    "toPageTop": "Retourner en haut"
+    "toPageTop": "Retourner en haut",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/fr.json:21]: @French translation team: Please add a **suggestion** here

File: src/assets/locales/fr.json [hunk 9]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/fr.json:22]: French translation team: Please add a **suggestion** here

File: src/assets/locales/fr.json [hunk 10]
```diff
@@ -195,7 +197,8 @@
           "subHeadline": "Changer ces paramètres peuvent prendre du temps jusqu'à ce qu'ils deviennent actif!",
           "userTitleLanguage": "Langue du titre",
           "userTitleLanguageHint": "Ce paramètre défini quel titre va être utilisé pour afficher le nom du média."
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/fr.json:201]: French translation team: Please add a **suggestion** here

File: src/assets/locales/id.json [hunk 11]
```diff
@@ -17,7 +17,9 @@
     "search": "Cari",
     "sort": "Urut & Saring",
     "toPageTop": "Lompat ke atas",
-    "filter": "Saring"
+    "filter": "Saring",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/id.json:21]: Indonesian translation team: Please add a **suggestion** here

File: src/assets/locales/id.json [hunk 12]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/id.json:22]: Indonesian translation team: Please add a **suggestion** here

File: src/assets/locales/id.json [hunk 13]
```diff
@@ -195,7 +197,8 @@
           "subHeadline": "Mengubah pengaturan ini bisa menghabiskan beberapa waktu hingga sampai berlaku!",
           "userTitleLanguage": "Bahasa Judul",
           "userTitleLanguageHint": "Pengaturan ini menentukan judul mana yang digunakan untuk menampilkan nama media."
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/id.json:201]: Indonesian translation team: Please add a **suggestion** here

File: src/assets/locales/it.json [hunk 14]
```diff
@@ -17,7 +17,9 @@
     "save": "Salva",
     "search": "Cerca",
     "sort": "Ordina",
-    "toPageTop": "Torna in cima"
+    "toPageTop": "Torna in cima",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/it.json:21]: Italian translation team: Please add a **suggestion** here

File: src/assets/locales/it.json [hunk 15]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/it.json:22]: Italian translation team: Please add a **suggestion** here

File: src/assets/locales/it.json [hunk 16]
```diff
@@ -195,7 +197,8 @@
           "subHeadline": "Potrebbe volerci un po' di tempo prima che i cambiamenti diventino definitivi!",
           "userTitleLanguage": "Lingua del titolo",
           "userTitleLanguageHint": "Questa impostazione definisce quale lingua verrà utilizzata per visualizzare i titoli dei media"
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/it.json:201]: Italian translation team: Please add a **suggestion** here

File: src/assets/locales/my.json [hunk 17]
```diff
@@ -17,7 +17,9 @@
     "search": "Cari",
     "sort": "Isih & Tapis",
     "toPageTop": "Lompat ke atas",
-    "filter": "Tapis"
+    "filter": "Tapis",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/my.json:21]: Malay translation team: Please add a **suggestion** here

File: src/assets/locales/my.json [hunk 18]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/my.json:22]: Malay translation team: Please add a **suggestion** here

File: src/assets/locales/my.json [hunk 19]
```diff
@@ -195,7 +197,8 @@
           "subHeadline": "Mengubah tetapan ini mungkin memerlukan sedikit masa sehingga ia dapat dilaksanakan!",
           "userTitleLanguage": "Bahasa Tajuk",
           "userTitleLanguageHint": "Tetapan ini menentukan tajuk mana yang digunakan untuk memaparkan nama media."
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/my.json:201]: Malay translation team: Please add a **suggestion** here

File: src/assets/locales/pt_BR.json [hunk 20]
```diff
@@ -17,7 +17,9 @@
     "save": "Salvar",
     "search": "Pesquisar",
     "sort": "Ordenar",
-    "toPageTop": "Ir para o topo"
+    "toPageTop": "Ir para o topo",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/pt_BR.json:21]: Portuguese translation team: Please add a **suggestion** here

File: src/assets/locales/pt_BR.json [hunk 21]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/pt_BR.json:22]: Portuguese translation team: Please add a **suggestion** here

File: src/assets/locales/pt_BR.json [hunk 22]
```diff
@@ -195,7 +197,8 @@
           "subHeadline": "Alterar essas configurações podem levar algum tempo até terem efeito!",
           "userTitleLanguage": "Idioma do Título",
           "userTitleLanguageHint": "Esta configuração define qual título é usado para exibir o nome da mídia."
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/pt_BR.json:201]: Portuguese translation team: Please add a **suggestion** here

File: src/assets/locales/tr.json [hunk 23]
```diff
@@ -17,7 +17,9 @@
     "save": "Kaydet",
     "search": "Ara",
     "sort": "Sırala",
-    "toPageTop": "En üste çık"
+    "toPageTop": "En üste çık",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/tr.json:21]: Turkish translation team: Please add a **suggestion** here

File: src/assets/locales/tr.json [hunk 24]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/tr.json:22]: Turkish translation team: Please add a **suggestion** here

File: src/assets/locales/tr.json [hunk 25]
```diff
@@ -195,7 +197,8 @@
           "scoringFormatHint": "Bu ayar animelerin nasıl puanlanacağını tanımlar.",
           "showExplicitContent": "Yetişkin içeriği göster (+18 içerik)",
           "showExplicitContentHint": "Bu ayar yetişkin içeriğin (+18) gösterilme durumunu tanımlar."
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/tr.json:201]: Turkish translation team: Please add a **suggestion** here

File: src/assets/locales/zh_CN.json [hunk 26]
```diff
@@ -17,7 +17,9 @@
     "search": "搜索",
     "sort": "排序",
     "toPageTop": "返回顶部",
-    "filter": "筛选"
+    "filter": "筛选",
+    "putOnRepeatList": "Put on repeat list",
```
Comment (NicoAiko) [inline src/assets/locales/zh_CN.json:21]: Chinese translation team: Please add a **suggestion** here

File: src/assets/locales/zh_CN.json [hunk 27]
```diff
@@ -17,7 +17,9 @@
...
+    "revertToAiringList": "Revert to airing list"
```
Comment (NicoAiko) [inline src/assets/locales/zh_CN.json:22]: Chinese translation team: Please add a **suggestion** here

File: src/assets/locales/zh_CN.json [hunk 28]
```diff
@@ -195,7 +197,8 @@
           "subHeadline": "更改这些设置可能需要一段时间才会生效！",
           "userTitleLanguage": "片名语言",
           "userTitleLanguageHint": "此项设置用于修改作品片名在显示时所使用的语言。"
-        }
+        },
+        "notLoggedIn": "You're currently not logged in! Please do so by clicking on the \"Login\" button below."
```
Comment (NicoAiko) [inline src/assets/locales/zh_CN.json:201]: Chinese translation team: Please add a **suggestion** here

File: src/assets/locales/tr.json (hunk 23 above)
Comment (xLn2) [inline src/assets/locales/tr.json:21]:
```suggestion
    "putOnRepeatList": "Tekrarlananlar listesine ekle",
```

File: src/assets/locales/tr.json (hunk 25 above)
Comment (xLn2) [inline src/assets/locales/tr.json:201]:
```suggestion
        "notLoggedIn": "Henüz giriş yapmadınız! Lütfen aşağıdaki \"Giriş\" tuşuna basarak giriş işleminizi tamamlayın."
```

File: src/assets/locales/tr.json (hunk 24 above)
Comment (xLn2) [reply src/assets/locales/tr.json:22]:
```suggestion
    "revertToAiringList": "İzleniyor listesine geri al"
```

File: src/assets/locales/tr.json (hunk 25 above)
Comment (xLn2) [reply src/assets/locales/tr.json:201]:
```suggestion
        "notLoggedIn": "Henüz giriş yapmadınız! Lütfen aşağıdaki \"Giriş\" tuşuna basarak işleminizi tamamlayın."
```

File: src/assets/locales/tr.json (hunk 23 above)
Comment (xLn2) [reply src/assets/locales/tr.json:21]:
```suggestion
    "putOnRepeatList": "Tekrarlananlar listesine ekle",
```

File: src/assets/locales/fr.json (hunk 9 above)
Comment (TheRaphael0000) [reply src/assets/locales/fr.json:22]: "toPageTop": "Retourner en haut",
    "putOnRepeatList": "Mettre dans la liste à revoir",
    "revertToAiringList": "Retourner à la liste en cours de diffusion"

File: src/assets/locales/fr.json (hunk 10 above)
Comment (TheRaphael0000) [reply src/assets/locales/fr.json:201]: "notLoggedIn": "Vous n'êtes actuellement pas connecté ! Veuillez-vous connectez en cliquant sur le bouton \"Connexion\" ci-dessous."

Comment (TheRaphael0000) [reply src/assets/locales/fr.json:201]:
```suggestion
        "notLoggedIn": "Vous n'êtes actuellement pas connecté ! Veuillez-vous connectez en cliquant sur le bouton \"Connexion\" ci-dessous."
```

File: src/assets/locales/fr.json (hunk 9 above)
Comment (TheRaphael0000) [reply src/assets/locales/fr.json:22]:
```suggestion
    "toPageTop": "Retourner en haut",
    "putOnRepeatList": "Mettre dans la liste à revoir",
    "revertToAiringList": "Retourner à la liste en cours de diffusion"
```

Comment (NicoAiko) [reply src/assets/locales/fr.json:22]: I'm not sure if we can input the lines all at once, but we'll try.

Comment (NicoAiko) [reply src/assets/locales/fr.json]: Thought so.
Now entries are doubled.
Please enter them one by one the next time.

File: src/assets/locales/pt_BR.json (hunk 21 above)
Comment (Sam-Oak) [inline src/assets/locales/pt_BR.json:22]:
```suggestion
    "revertToAiringList": "Reverter para lista de exibição"
```

File: src/assets/locales/pt_BR.json (hunk 22 above)
Comment (Sam-Oak) [inline src/assets/locales/pt_BR.json:201]:
```suggestion
        "notLoggedIn": "Você não está logado no momento! Por favor faça isso clicando no botão \"Login\" abaixo."
```

**Review by Sam-Oak — CHANGES_REQUESTED** [review]

File: src/assets/locales/ar.json (hunk 1 above)
Comment (CeStain) [inline src/assets/locales/ar.json:21]:
```suggestion
    "putOnRepeatList": "وضع في قائمة التكرار",
```

File: src/assets/locales/ar.json (hunk 2 above)
Comment (CeStain) [inline src/assets/locales/ar.json:22]:
```suggestion
    "revertToAiringList": "العوده لقائمة البث"
```

File: src/assets/locales/ar.json (hunk 3 above)
Comment (CeStain) [inline src/assets/locales/ar.json:201]:
```suggestion
        "notLoggedIn": "انت حالياً غير مسجل! يرجى تسجيل الدخول عن طريق الضغط علي زر التسجيل في الأسفل."
```

**Review by CeStain — APPROVED** [review]

**Review by CeStain — DISMISSED** [review]

File: src/assets/locales/id.json (hunk 12 above)
Comment (nattadasu) [inline src/assets/locales/id.json:22]:
```suggestion
    "putOnRepeatList": "Masukkan ke daftar ulang",
    "revertToAiringList": "Kembalikan ke daftar mengudara"
```

File: src/assets/locales/id.json (hunk 13 above)
Comment (nattadasu) [inline src/assets/locales/id.json:201]:
```suggestion
        "notLoggedIn": "Kamu belum masuk! Untuk masuk, silakan tekan tombol \"Masuk\" di bawah ini."
```

File: src/assets/locales/my.json (hunk 18 above)
Comment (nattadasu) [inline src/assets/locales/my.json:22]:
```suggestion
    "putOnRepeatList": "Masukkan ke senarai ulangan",
    "revertToAiringList": "Kembalikan ke senarai siaran"
```

File: src/assets/locales/my.json (hunk 19 above)
Comment (nattadasu) [inline src/assets/locales/my.json:201]:
```suggestion
        "notLoggedIn": "Anda belum log masuk! Sila lakukan dengan mengklik pada butang \"Masuk\" di bawah."
```

**Review by nattadasu — CHANGES_REQUESTED** [review]
*Localizes Indonesian and Malay lines*

File: src/assets/locales/my.json (hunk 18 above)
Comment (nattadasu) [reply src/assets/locales/my.json:22]: Hold on, lemme fix that real quick

File: src/assets/locales/it.json (hunk 16 above)
Comment (Knocks83) [inline src/assets/locales/it.json:201]:
```suggestion
        "notLoggedIn": "Non sei loggato! Accedi usando il bottone \"Accedi\"."
```

File: src/assets/locales/it.json (hunk 15 above)
Comment (Knocks83) [inline src/assets/locales/it.json:22]:
```suggestion
    "revertToAiringList": "Reinseriscilo nella lista degli anime che stai guardando"
```

File: src/assets/locales/it.json (hunk 14 above)
Comment (Knocks83) [inline src/assets/locales/it.json:21]:
```suggestion
    "putOnRepeatList": "Inseriscilo nella lista degli anime che stai riguardando",
```

**Review by Knocks83 — CHANGES_REQUESTED** [review]
Fixed IT translation

File: src/assets/locales/ja.json [hunk 29]
```diff
@@ -17,7 +17,9 @@
     "save": "保存",
     "search": "検索",
     "sort": "並べる",
-    "toPageTop": "トップへ"
+    "toPageTop": "トップへ",
+    "putOnRepeatList": "リピートリストに移動する",
+    "revertToAiringList": "「見る」リストに戻す"
```
Comment (NicoAiko) [inline src/assets/locales/ja.json:22]:
```suggestion
    "revertToAiringList": "「視聴中」リストに戻す"
```

File: src/assets/locales/ja.json [hunk 30]
```diff
@@ -17,7 +17,9 @@
     "save": "保存",
     "search": "検索",
     "sort": "並べる",
-    "toPageTop": "トップへ"
+    "toPageTop": "トップへ",
+    "putOnRepeatList": "リピートリストに移動する",
```
Comment (NicoAiko) [inline src/assets/locales/ja.json:21]:
```suggestion
    "putOnRepeatList": "再視聴中リストに移動する",
```

**Pull request merged**